### PR TITLE
Respect standard notations for images in functions arguments

### PR DESCRIPTION
**Pull request by sciunto**
## Description

**ready for reviews**

See #2616 and #2538

* img -> image
* im -> image

### CODE REVIEW

**Comment by sciunto** [comment]
Note: I changed the API of a private function for chan vese (_cv_init_level_set), images were passed but on the shape was used. Now, I pass only the shape.

The rest is supposed to be only a bunch of substitutions.

**Comment by sciunto** [comment]
@soupault congrat's, you arrived at the end :)

File: doc/examples/color_exposure/plot_local_equalize.py [hunk 1]
```diff
@@ -34,27 +34,27 @@
 matplotlib.rcParams['font.size'] = 9
 
 
-def plot_img_and_hist(img, axes, bins=256):
+def plot_img_and_hist(image, axes, bins=256):
```
Comment (jni) [inline doc/examples/color_exposure/plot_local_equalize.py]: @sciunto how about deprecating this in favour of `plot_image_and_histogram`? =) It's easy with our `deprecated` decorator!

Comment (jni) [inline doc/examples/color_exposure/plot_local_equalize.py]: Oh I just saw that this is an example! LOL well then it's even easier! =)

File: skimage/segmentation/_chan_vese.py [hunk 2]
```diff
@@ -150,24 +150,24 @@ def _cv_small_disk(image_size):
     return (radius-distance(res)) / (radius*3)
 
 
-def _cv_init_level_set(init_level_set, img):
+def _cv_init_level_set(init_level_set, img_shape):
```
Comment (jni) [inline skimage/segmentation/_chan_vese.py]: I feel like this rename should be `image_shape` or just `shape`

File: skimage/transform/seam_carving.py [hunk 3]
```diff
@@ -47,18 +47,18 @@ def seam_carve(img, energy_map, mode, num, border=1, force_copy=True):
...
-    utils.assert_nD(img, (2, 3))
-    image = util.img_as_float(img, force_copy)
+    utils.assert_nD(image, (2, 3))
+    img = util.img_as_float(image, force_copy)
     energy_map = util.img_as_float(energy_map, force_copy)
 
-    if image.ndim == 2:
-        image = image[..., np.newaxis]
+    if img.ndim == 2:
+        img = img[..., np.newaxis]
```
Comment (jni) [inline skimage/transform/seam_carving.py]: Erm.... What??? LOL

File: skimage/transform/seam_carving.py [hunk 4]
```diff
@@ -47,18 +47,18 @@ def seam_carve(img, energy_map, mode, num, border=1, force_copy=True):
            http://www.cs.jhu.edu/~misha/ReadingSeminar/Papers/Avidan07.pdf
     """
 
-    utils.assert_nD(img, (2, 3))
-    image = util.img_as_float(img, force_copy)
+    utils.assert_nD(image, (2, 3))
+    img = util.img_as_float(image, force_copy)
```
Comment (jni) [inline skimage/transform/seam_carving.py]: @sciunto do you want to use this PR to deprecate `img_as_*` in favour of `image_as_*`? Your call if you want to leave for later PR. But you should definitely revert this `img` back to `image`. Dunno what happened there. =)

Comment (soupault) [inline skimage/transform/seam_carving.py]: @jni I don't think that it's worthwhile to do this. We have `rescale_to_*` on its way (https://github.com/scikit-image/scikit-image/issues/1234).

Comment (jni) [inline skimage/transform/seam_carving.py]: @soupault Ah! I forgot about that one! Thanks for the reminder!

Comment (jni) [inline skimage/transform/seam_carving.py]: That still leaves changing the assigned name to `image`, though. =)

**Comment by sciunto** [comment]
@jni It's done! (I was tricked by the force_copy, that's why I changed the variable name).

**Pull request merged**